module/VuFindApi/src/VuFindApi/Controller/SearchApiController.php

```diff
@@ -170,6 +171,8 @@ public function __construct(
                 $this->$key = $settings[$key];
             }
         }
+        $config = $this->getConfigArray('config')['API_Keys'] ?? [];
```

Comment:
You can use the default value here to shorten this:

```suggestion
        $config = $this->getConfigArray()['API_Keys'] ?? [];
```